### ISSUE TITLE
[COGNITION]: Focus should be set on body tag when not in app flow

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6235]
## Description

We will want to leave focus on the `<body>` tag when we're not in app flow. This is a related ticket to #6070 . @jbalboni asked to split them so he could focus on the two related, but unique, challenges. Screenshots attached for situations where we want to set focus back on the `<body>` tag.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/6070

## Acceptance Criteria
**Definition of done**
* [ ] As a screen reader user, I want to hear the original VAOS app `<title>` read aloud
* [ ] I should be able to confirm the body has focus with `$ document.activeElement`

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-20 at 4 44 45 PM](https://user-images.githubusercontent.com/934879/74987526-fc6daa00-5400-11ea-8513-a4d6eac1bfb4.png)

---
![Screen Shot 2020-02-26 at 1 50 37 PM](https://user-images.githubusercontent.com/934879/75382057-6bcf1800-589f-11ea-82df-b22ca61a6da0.png)

---
![Screen Shot 2020-02-26 at 1 50 43 PM](https://user-images.githubusercontent.com/934879/75382071-712c6280-589f-11ea-86f2-bb6914fa3c49.png)

**Comment by jbalboni** [comment]
The problem I've run into is that Chrome seems to try to restore your tab position at a weird spot, because the loading indicator component sets focus to itself while the profile info is loading.

This is easy enough to turn off, then Chrome/Firefox/Safari seem consistent (except FF doesn't open the Skip to Content link). But it made me wonder if this is the right choice. Because this is a SPA, the main content will likely load well after the header/footer for a significant number of users. What happens to a user if they skip to the main content and see the loading indicator? Is resetting them to the top of the page again the right choice? None of the header content has changed and I could see it being confusing.

**Comment by 1Copenut** [comment]
@lenaecb After talking more with @jbalboni we realized this ticket is not needed. Our issue #6070 captures the focus management need completely. I'm going to close this one.
